homeassistant/components/camera/prefs.py

```diff
@@ -16,28 +17,19 @@
 STORAGE_VERSION: Final = 1
 
 
-class CameraEntityPreferences:
-    """Handle preferences for camera entity."""
+@dataclass
+class DynamicStreamSettings:
+    """Stream settings which are managed and updated by the camera entity."""
 
-    def __init__(self, prefs: dict[str, bool | Orientation]) -> None:
-        """Initialize prefs."""
-        self._prefs = prefs
+    preload_stream: bool
```

Comment:
I see below there is some use of dynamic stream settings with a constructor for default values. Want to set those here as default values?